libres/lib/analysis/ies/ies_enkf_data.cpp

```diff
@@ -248,18 +248,14 @@ void ies_enkf_data_store_initialA(ies_enkf_data_type *data,
     }
 }
 
-void ies_enkf_data_allocateW(ies_enkf_data_type *data, int ens_size) {
+void ies_enkf_data_allocateW(ies_enkf_data_type *data) {
     if (!data->W) {
         // We initialize data-W which will store W for use in next iteration                    (Line 9)
         bool dbg = ies_enkf_config_get_ies_debug(data->config);
+        int ens_size = bool_vector_size(data->ens_mask);
         int m_ens_size = std::min(ens_size - 1, 16);
```

Comment:
Not related to this PR, but why 16?

Reply:
`m_ens_size` is not in use though, so 16 doesn't matter :)

Reply:
> Not related to this PR, but why 16?

It is a variety of 80 characters on a line ... purely visual formatting.

Reply:
I think the important part here is that this is an unused variable so perhaps have another look at where this should go or whether it should be simply removed.

Reply:
Everything removed